File: Python/ToLowerCaseTest.py
from unittest import TestCase

from ToLowerCase import ToLowerCase


class ToLowerCaseTest(TestCase):
    def test_toLowerCase(self):
        tlc = ToLowerCase()

        self.assertEqual(tlc.toLowerCase("Hello"), "hello")
        self.assertEqual(tlc.toLowerCase("here"), "here")
        self.assertEqual(tlc.toLowerCase("LOVELY"), "lovely")

File: Golang/ToLowerCase/ToLowerCase_test.go
package ToLowerCase

import "testing"

func TestToLowerCase(t *testing.T) {
	if toLowerCase("Hello") == "hello" {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

	if toLowerCase("here") == "here" {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

	if toLowerCase("LOVELY") == "lovely" {
		t.Log("pass")
	} else {
		t.Error("failed")
	}
}
